package.json

```diff
@@ -62,37 +62,15 @@
 		"rn-bundle": "react-native bundle",
 		"postrn-bundle": "patch-package --reverse --patch-dir gutenberg/packages/react-native-editor/metro-patch",
 		"i18n:update": "./bin/i18n-update.sh jetpack ./jetpack/projects/plugins/jetpack/extensions layout-grid ./block-experiments/blocks/layout-grid",
-		"i18n:update:debug": "./bin/i18n-update.sh --debug jetpack ./jetpack/projects/plugins/jetpack/extensions layout-grid ./block-experiments/blocks/layout-grid",
-		"i18n:gutenberg": "./bin/i18n-download.sh gutenberg",
-		"i18n:gutenberg:force": "cross-env REFRESH_I18N=1 ./bin/i18n-download.sh gutenberg",
-		"bundle": "npm run clean; npm run bundle:js && npm run genstrings",
-		"prebundle:js": "npm run i18n:gutenberg:force",
+		"i18n:update:skip-if-cache": "./bin/i18n-update.sh --skip-if-cache jetpack ./jetpack/projects/plugins/jetpack/extensions layout-grid ./block-experiments/blocks/layout-grid",
+		"i18n:update:test": "./bin/i18n-update.sh --path i18n-test jetpack ./jetpack/projects/plugins/jetpack/extensions layout-grid ./block-experiments/blocks/layout-grid",
```

Comment:
This is an extra command I added in case we want to check the intermediate files produced when updating localizations.